.changeset/slick-lamps-clap.md

```diff
@@ -0,0 +1,11 @@
+---
+'@backstage/plugin-catalog-backend-module-msgraph': minor
+---
+
+Encode query filters for requests made to msgraph. If you currently have manually encoded characters in a filter, this is a breaking change and must be updated to avoid requests being double encoded.
```

Comment:
```suggestion
**BREAKING**: Encode query filters for requests made to msgraph. If you currently have manually encoded characters in a filter, this is a breaking change and must be updated to avoid requests being double encoded.
```

this makes it easier to discover